### ISSUE TITLE
Rule S5725 Disabling resource integrity features is security-sensitive

### ISSUE THREAD

**Issue by eric-therond-sonarsource** [SonarSource/SonarJS#1944]
Implements [RSPEC-5725](https://jira.sonarsource.com/browse/RSPEC-5725)

For both JS/TS

Code examples: https://github.com/SonarSource/security-expected-issues/tree/master/javascript/rules/hotspots/RSPEC-5725

```js
var script = document.createElement("script"); // Sensitive
script.src = "https://cdnexample.com/script-latest.js";
script.crossOrigin = "anonymous";
document.head.appendChild(script);
```

Report on `document.createElement("script")` when for resulting element following conditions are met:
* `script.src` is set to a value starting with "http" or "//"
* `script.integrity` is not defined
